python/lsst/ap/verify/testPipeline.py

```diff
@@ -433,6 +434,16 @@ def run(self, science, matchedTemplate, difference,
                       )
 
 
+class MockRBTransiNetTask(PipelineTask):
+    """A do-nothing substitute for RBTransiNetTask.
+    """
+    _DefaultName = "rbTransiNet"
```

Comment:
The value of `_DefaultName` is missing a `not` prefix. Looking at other examples, I'm assuming it should be called `notRbTransiNet` or something along those lines.